.github/workflows/build.yml

```diff
@@ -52,12 +52,11 @@ jobs:
       matrix:
         image:
           - { name: "ubuntu", tag: "20.04" }
-        binja:
-          - { channel: "headless", version: "3.1.3479" }
-        llvm: ["14", "15"]
-        cxxcommon_version: ["v0.2.16"]
+        llvm: ["15"]
+        cxxcommon_version: ["v0.2.22"]
 
-    runs-on: ubuntu-20.04
+    runs-on:
+      labels: gha-ubuntu-32
```

Comment:
A larger runner is necessary?

Reply:
we have like almost 200 test binaries that need to run through (and a bunch more that are disabled) so i figured using a larger one would be better for this case